.github/release.yml

```diff
@@ -0,0 +1,53 @@
+# This workflow is useful if you want to automate the process of:
+#
+# a) Creating a new prelease when you push a new tag with a "v" prefix (version).
```

Comment:
pre-release?

Reply:
I didn’t get you.

Reply:
There is a typo error in the sentence. prelease --> pre-release

Reply:
also if you can change all usage of prerelease --> pre-release that will be great.

Reply:
Sure